test/Chirp.Infrastructure.Tests/CheepRepositoryCreateTest.cs

```diff
@@ -45,19 +45,20 @@ public CheepRepositoryUnitTests()
     }
 
     [Fact]
-    public async Task UnitTestCreateMethod()
+    public async void UnitTestCreateMethod()
     {
         //Arrange
         string Message = "TestMessage";
         CheepCreateDTO cheepCreateDto = new CheepCreateDTO("TestAuthor", Message);
 
         //Act
-        Cheep result = await repository.Create(cheepCreateDto);
+        await repository.Create(cheepCreateDto); //Adds the cheep to the database
+
+         var cheeps = repository.GetCheeps(1);
+        cheeps.Should().NotBeNull(); //Makes sure the page is not empty
 
         //Assert
-        Assert.NotNull(result);
-        Assert.Equal("TestAuthor", result.Author.Name);
-        Assert.Equal("TestMessage", result.Text);
+        cheeps.Should().Contain(c => c.Author == "TestAuthor" && c.Message == "TestMessage");
```

Comment:
Just curious; does the method .Should() work as an assertion?
And will the dotnet test command evaluate on it?

Reply:
Its fluent assertions, so not completely. You can always look at it on the official package page. But it works more like how we think as a person. We say that it should either contain or not contain, or that it should throw or not throw. It used in a lot of other tests.

It works with the dotnet test command as intended if thats what you mean?

Reply:
Nice good to know, thanks.